README.md

```diff
@@ -101,45 +101,51 @@ Please see [this file](./__tests__/README.md) for more info.
 ###### Mentor intake schema:
 
     {
-        "profile_id": '00u13oned0U8XP8Mb4x7',
-        "email": 'fakeemail2@gmail.com',
-        "country": 'USA',
-        "state": 'California',
-        "city": 'San Francisco',
-        "first_name": 'Hotdog',
-        "last_name": 'Jeopardy',
-        "current_company": 'Amazin',
-        "current_position": 'Front End Developer',
+        "profile_id": "string",
+        "first_name": "string",
+        "last_name": "string",
+        "email": "user@example.com",
+        "country": "string",
+        "state": "string",
+        "city": "string",
+        "current_company": "string",
+        "current_position": "string",
+        "tech_stack": [
+            "string"
+        ],
         "job_help": true,
         "industry_knowledge": true,
-        "pair_programming": false,
-        "tech_stack": 'frontend'
-        "experience_level": 'expert',
-        "other_info": 'none',
-        "validateStatus: 'pending',
+        "pair_programming": true,
+        "commitment": true,
+        "referred_by": "string",
+        "other_info": "string",
+        "validate_status": "pending",
+        "is_active": false,
+        "accepting_new_mentees": false
     }
 
 ###### Mentee intake schema:
 
     {
-        "profile_id": '12',
-        "email": 'fakeemail1@gmail.com',
-        "country": 'USA',
-        "state": 'California',
-        "city": 'San Francisco',
-        "first_name": 'Joe',
-        "last_name": 'Baseball',
+        "profile_id": "string",
+        "first_name": "string",
+        "last_name": "string",
+        "email": "user@example.com",
+        "country": "string",
+        "state": "string",
+        "city": "string",
         "formerly_incarcerated": true,
-        "list_convictions": 'coded too much',
```

Comment:
This is a serious infraction, I'm not sure we should have deleted it. But overall the application is stronger and more robust, well done.

Reply:
I did updated it with the schema in the DS API app. the information here is from the deprecated seed. I figured adding what the DS API is actually expecting will be more helpful since it matches what is on AWS.